R/convert_handcrafted_questionnaire_to_mdm_format.R

```diff
@@ -206,7 +206,7 @@ trim_list_cols <- function(excel, ...) {
 }
 
 list_attribute <- function(str_attr) {
-  attr <- unlist(strsplit(str_attr, ",", fixed = TRUE))
+  attr <- unlist(strsplit(str_attr, "[,;-]+", fixed = FALSE))
```

Comment:
@rbirkelbach You cannot split by "-" cause this is a valid character in question numbers. See https://metadatamanagement.readthedocs.io/de/stable/javadoc/eu/dzhw/fdz/metadatamanagement/questionmanagement/domain/Question.html#number